### PR TITLE
Upgrade Website

### PR DESCRIPTION
**Pull request by Thomas-Boi**
# **New feature section**

## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *New Feature: a brief description of feature*

## This PR adds/fixes

Updated/add a couple features to the website:
- Added a color picker, which addresses #943.
- Move the instruction to paste the `<link>` tag up. Addresss #971 
- Add a copy to clipboard button. Addresses #968 
- Add a contact email and clarify some legal stuff. Addresses #972

## Notes

@Alirezaaraby @Snailedlt @amacado please check out the PR and let me know if it addresses your problems.

### CODE REVIEW

File: docs/assets/css/style.css [hunk 1]
```diff
@@ -403,15 +462,9 @@ header {
...
   margin-top: 6rem;
-  text-align: center; }
-
-input[type='color'] {
-  width: 1.25em;
-  height: 1.25em;
-  margin-left: 0.5em;
-}
\ No newline at end of file
+  text-align: center; }
```
Comment (Panquesito7) [inline docs/assets/css/style.css:470]: Missing an endline.
```suggestion
  text-align: center; }

```

File: docs/index.html [hunk 2]
```diff
@@ -57,81 +57,146 @@ <h1>To use this website, JavaScript needs to be enabled.</h1>
     <header class="clearfix">
       <h1>Devicon <span><a href="https://github.com/devicons/devicon/releases">{{ latestReleaseTagging }}</a></span></h1>
       <h2>Devicon is a set of icons representing programming languages, designing & development tools. You can use it as a font or directly copy/paste the svg code into your project.</h2>
```
Comment (Panquesito7) [inline docs/index.html:59]:
```suggestion
      <h2>Devicon is a set of icons representing programming languages, designing, and development tools. You can use it as a font or directly copy/paste the SVG code into your project.</h2>
```

File: docs/index.html [hunk 3]
```diff
@@ -57,81 +57,146 @@ <h1>To use this website, JavaScript needs to be enabled.</h1>
...
+      <i>*Please read the legal conditions of using the logos here at the bottom of this header</i>
```
Comment (Panquesito7) [inline docs/index.html:60]: Is this `*` added intentionally or is it a typo?

File: docs/index.html [hunk 4]
```diff
@@ -57,81 +57,146 @@ <h1>To use this website, JavaScript needs to be enabled.</h1>
...
         <a href="https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md" target="blank"><i class="devicon-github-original"></i>CONTRIBUTE</a>
       </p>
-      <p class="download">
+      <p class="buttonLink">
         <a href="https://discord.gg/hScy8KWACQ" target="blank"><i class="discord-logo"></i>DISCORD SERVER</a>
       </p>
 
-      <p class="footer">
+      <h3>Contact</h3>
+      <h5>If you have any legal concerns regarding copyrights, please reach out to us at <b><a>info@devicon.dev</a></b>. Any code/logo contributions should be made through our GitHub repo listed above.</h5>
```
Comment (Panquesito7) [inline docs/index.html:189]:
```suggestion
      <h5>If you have any legal concerns regarding copyrights, please reach out to us at <b><a>info@devicon.dev</a></b>. Any code/logo contributions should be made through our GitHub repository listed above.</h5>
```

File: docs/index.html (hunk 3 above)
Comment (Thomas-Boi) [reply docs/index.html:60]: It's intentional. For me, it works as an attention grabber but we can remove it if you want.

**Comment by Snailedlt** [comment]
I don't think I'll have time to look at this until next year. But without looking at the code, the bullet points look very promising :)

**Comment by Alirezaaraby** [comment]
Good. The #968 solved but i didn't find any commit message pointed to #971.

**Comment by Thomas-Boi** [comment]
> Good. The #968 solved but i didn't find any commit message pointed to #971.

It's not included in its own commit. I included in with one of the other commits. If you check out the website through the last commit in the PR, you should see the "Place this in your header" section moved up above the icons themselves.

**Comment by amacado** [comment]
Thanks for picking those issues up @Thomas-Boi! I've looked at it and just have some minor improvements in #1024. When you like it you can accept it and merge it in this branch! 😄

**Comment by Thomas-Boi** [comment]
@amacado let me know if you need anything else for this PR. I think this should be fine now.

**Review by amacado — APPROVED** [review]

**Pull request merged**